notebooks/source/DNS_Analytics.py

```diff
@@ -8,9 +8,14 @@
 
 # COMMAND ----------
 
-# MAGIC %python
-# MAGIC #Set this flag to false if you are not running this notebook in community edition
-# MAGIC community_edition = False
+# MAGIC %pip install tldextract
+# MAGIC %pip install geoip2
+# MAGIC %pip install dnstwist
```

Comment:
it's better to have all packages installed with one command, otherwise it will restart Python interpreter multiple times: `%pip install tldextract geoip2 dnstwist`